packages/runtime/src/widgets/Search.tsx

```diff
@@ -183,6 +185,10 @@ export const Search: React.FC<SearchProps> = ({
     [extractValue, namedData, rest.selectedLabel],
   );
 
+  useEffect(() => {
+    setValue(values?.initialValue);
+  }, [values?.initialValue]);
```

Comment:
This should only set the value if there is no existing value already.

Reply:
@evshi there will be no value on first render every time and due to initialValue being a constant string, it won't run again.
but if i add a check for `value` in it, it will check it every time the `value` gets updated. correct me if im wrong